uvm/js/common/ungrid/GridController.js

```diff
@@ -440,6 +440,68 @@ Ext.define('Ung.cmp.GridController', {
         var grid = this.getView(),
             existingData = Ext.Array.pluck(grid.getStore().getRange(), 'data');
 
+        var validData = [];
+        var validationErrors = [];
+        newData.forEach(function(record) {
+                var isValidRecord = true;
+                for (var fieldName in record) {
+                    if (record.hasOwnProperty(fieldName)) {
+                        var fieldValue = record[fieldName];
+                        var fieldConfig = this.getFieldConfig(fieldName);
+        
+                        if (fieldConfig !== undefined && (fieldConfig.validator || fieldConfig.vtype)) {
+                            var validationErrorMsg = null;
+        
+                            if (!fieldConfig.allowBlank && !fieldConfig.bind.disabled && Ext.isEmpty(fieldValue)) {
+                                validationErrorMsg = 'This field is required.'; 
+                            } else if (fieldConfig.allowBlank && Ext.isEmpty(fieldValue)) {
+                                continue; // Skip validation if allowBlank is true and field value is empty
+                            }
+        
+                            // Check vtype validation
+                            if (fieldConfig.vtype) {
+                                var vtype = fieldConfig.vtype;
+                               if(fieldValue !== null){
+                                    if (!Ext.form.field.VTypes[vtype](fieldValue)) {
+                                        validationErrorMsg = Ext.form.field.VTypes[vtype + 'Text'];
+                                    }
+                               }
+                            }
+        
+                            // Check custom validator 
+                            if(record.javaClass === 'com.untangle.app.wireguard_vpn.WireGuardVpnTunnel'){
```

Comment:
You said you were not comfortable doing this globally for other validator functions?  If so, please expand the comment to explain.

Reply:
Updated the comment. Also we can create JIRA to do the required changes for each apps to check custom validators in case of import operation.